compute/client_library/snippets/disks/resize_disk.py

```diff
@@ -0,0 +1,122 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# flake8: noqa
+
+
+# This file is automatically generated. Please do not modify it directly.
+# Find the relevant recipe file in the samples/recipes or samples/ingredients
+# directory and apply your changes there.
+
+
+# [START compute_regional_disk_resize]
+# [START compute_disk_resize]
+import re
+import sys
+from typing import Any, NoReturn
+
+from google.api_core.extended_operation import ExtendedOperation
+from google.cloud import compute_v1
+
+
+def wait_for_extended_operation(
```

Comment:
It seems like this code is duplicated in other files as well - could you refactor this to like a utility function instead?

Reply:
Yes, this code is often repeated, because I always want to provide a demonstration on how to handle the operation object and its results. It would be nice to turn it into an utility function of the Python GAPIC library, but that would be a bigger task that I currently don't even know how to start :)

Reply:
Ah, just an utility function within `compute/client_library/snippets/util.py` or something would work! Doesn't have to be back in the GAPIC library.

Reply:
But then it wouldn't be part of the sample and people would wonder where is the code for this. I want all the samples to be available as copy-paste pieces that will work, without additional research required from developer to find this one utility function.

Reply:
Ah, I confused myself thinking that you could import the same region tag over multiple files - you can do it over the same file but not over different files as it'll cause a sync issue. SGTM to keep it as is, the tradeoff for user experience is better to have than maintenance over O(5) occurrences.